entwatch/ze_mapeadores_museum_test4.cfg

```diff
@@ -0,0 +1,231 @@
+        "shortname"         "Pulverizer"
+        "color"             "{red}"
+        "buttonclass"       "func_button"
+        "filtername"        ""
+        "hasfiltername"     "false"
+        "blockpickup"       "false"
+        "allowtransfer"     "true"
+        "forcedrop"         "true"
+        "chat"              "true"
+        "hud"               "true"
+        "hammerid"          "422468"
+        "mode"              "3"
+        "maxuses"           "1"
+        "cooldown"          "0"
+        "maxamount"         "8"
+    }
+    "6"
+    {
+        "name"              "Saw"
+        "shortname"         "Saw"
+        "color"             "{orange}"
+        "buttonclass"       ""
+        "filtername"        ""
+        "hasfiltername"     "false"
+        "blockpickup"       "false"
+        "allowtransfer"     "true"
+        "forcedrop"         "true"
+        "chat"              "true"
+        "hud"               "true"
+        "hammerid"          "422614"
+        "mode"              "0"
+        "maxuses"           "0"
+        "cooldown"          "0"
+        "maxamount"         "8"
+    }
+    "7"
+    {
+        "name"              "Magma Push"
+        "shortname"         "Magma Push"
+        "color"             "{red}"
+        "buttonclass"       "func_button"
+        "filtername"        ""
+        "hasfiltername"     "false"
+        "blockpickup"       "false"
+        "allowtransfer"     "true"
+        "forcedrop"         "true"
+        "chat"              "true"
+        "hud"               "true"
+        "hammerid"          "30745"
+        "mode"              "2"
+        "maxuses"           "0"
+        "cooldown"          "40"
+        "maxamount"         "2"
+    }
+    "8"
+    {
+        "name"              "ZM Bulldozer"
+        "shortname"         "ZM Bulldozer"
+        "color"             "{orange}"
+        "buttonclass"       ""
+        "filtername"        ""
+        "hasfiltername"     "false"
+        "blockpickup"       "false"
+        "allowtransfer"     "false"
+        "forcedrop"         "false"
+        "chat"              "true"
+        "hud"               "true"
+        "hammerid"          "422989"
+        "mode"              "0"
+        "maxuses"           "0"
+        "cooldown"          "0"
+        "maxamount"         "8"
+    }
+    "9"
+    {
+        "name"              "ZM Slasher"
+        "shortname"         "ZM Slasher"
+        "color"             "{darkorange}"
+        "buttonclass"       "func_button"
+        "filtername"        ""
+        "hasfiltername"     "false"
+        "blockpickup"       "false"
+        "allowtransfer"     "false"
+        "forcedrop"         "false"
+        "chat"              "true"
+        "hud"               "true"
+        "hammerid"          "423040"
+        "mode"              "1"
+        "maxuses"           "0"
+        "cooldown"          "8"
+        "maxamount"         "8"
+    }
+    "10"

```

Comment:
I would **highly** suggest you include `"trigger"` for the zombie items so restricted players don't walk over knives and get their knives stripper but not pick up items.

Reply:
Ah so thats what its for. I’ll add it later today, thanks for letting me know.